src/registrar/views/application.py

```diff
@@ -266,6 +285,38 @@ def get_forms(self, step=None, use_post=False, use_db=False, files=None):
 
         return instantiated
 
+    def pending_requests(self):
+        """return an array of pending requests if user has pending requests
+        and no approved requests"""
+        if self.approved_applications_exist() or self.approved_domains_exist():
+            return []
+        else:
+            return self.pending_applications()
+
+    def approved_applications_exist(self):
+        """Checks if user is creator of applications with APPROVED status"""
+        approved_application_count = DomainApplication.objects.filter(
+            creator=self.request.user, status=DomainApplication.APPROVED
+        ).count()
+        return approved_application_count > 0
+
+    def approved_domains_exist(self):
+        """Checks if user has permissions on approved domains
+
+        This additional check is necessary to account for domains which were migrated
+        and do not have an application"""
+        return self.request.user.permissions.count() > 0
+
+    def pending_applications(self):
+        """Returns a List of user's applications with one of the following states:
+        SUBMITTED, IN_REVIEW, ACTION_NEEDED"""
+        # if the current application has ACTION_NEEDED status, this check should not be performed
+        if self.application.status == DomainApplication.ACTION_NEEDED:
+            return []
+        check_statuses = [DomainApplication.SUBMITTED, DomainApplication.IN_REVIEW, DomainApplication.ACTION_NEEDED]
+        filter_conditions = Q(creator=self.request.user) & Q(status__in=check_statuses)
+        return DomainApplication.objects.filter(filter_conditions)
```

Comment:
```suggestion
    return DomainApplication.objects.filter(
        creator=self.request.user,
        status__in=check_statuses
    )
```
While this works, I think using Q might be a bit overkill here. The filter above should be syntactically equivalent (though I wouldn't commit that directly, the spacing is broke on the github end).

I would recommend this just for legibility purposes